### ISSUE TITLE
Support latex?

### ISSUE THREAD

**Issue by AdhTri001** [realdennis/md2pdf#77]
can we have a support for latex math equations? Like the [pandoc markdown](https://pandoc.org/MANUAL.html#pandocs-markdown)

**Comment by RedGrapefruit09** [comment]
Yeah, for example to render stuff like: $\frac{1}{2}$. I'm an Obsidian.md user and I use md2pdf to export and print my notes, so it'd be really helpful to have this!

**Comment by alialili** [comment]
Is that possible to add this support?
